_includes/alert.html

```diff
@@ -0,0 +1,12 @@
+{% comment %}
+include
+- content
+- type (optional)
+{% endcomment %}
+<div class="usa-alert usa-alert--{{ include.type | default: 'info' }}">
+  <div class="usa-alert__body">
+    <p class="usa-alert__text">
```

Comment:
the `| markdownify` didn't get copied over, do we want `markdown="1"` here?
```suggestion
    <p class="usa-alert__text" markdown="1">
```

Reply:
I was semi-intentional in dropping the `markdownify`, since I didn't want to impose it as a requirement that the paragraph tag be provided by the parent view, and calling `markdownify` introduces a new paragraph that can't be nested here.

We only use `create-an-account.html` in one place, which has a single `info` value that does not include Markdown.

https://github.com/18F/identity-site/blob/f49bf361bf17038d266e961eb4b51c82a5b4422f/content/_landing/create_an_account._en.md#L34-L36

Reply:
Makes sense!